src/store/blockstore-level.ts

```diff
@@ -45,7 +45,8 @@ export class BlockstoreLevel implements Blockstore {
   }
 
   async get(key: CID | string, options?: Options): Promise<Uint8Array> {
-    return this.db.get(String(key), options);
+    const result = await this.db.get(String(key), options);
+    return result!;
```

Comment:
Instead of `result!`, let's update return type to `Promise<Uint8Array | undefined>

Reply:
Done.

Reply:
Fixing this requires updating the `interface-blockstore` package.